01_Intro/Variable/Variable/Program.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Variable
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {  //videourl : https://mva.microsoft.com/en-US/training-courses/c-fundamentals-for-absolute-beginners-16169?l=Y6D7PQQIC_5406218949
+            /*  int x;
```

Comment:
Auskommentierter code ist in der regel immer verwirrender code. Entweder entfernen oder wieder richtigen code daraus machen.